helm/superset/templates/deployment.yaml

```diff
@@ -117,6 +117,24 @@ spec:
             - name: http
               containerPort: {{ .Values.service.port }}
               protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: http
+            initialDelaySeconds: {{ .Values.supersetNode.livenessProbe.initialDelaySeconds | default 15 }}
+            timeoutSeconds: {{ .Values.supersetNode.livenessProbe.timeoutSeconds | default 1 }}
+            failureThreshold: {{ .Values.supersetNode.livenessProbe.failureThreshold | default 3 }}
+            periodSeconds: {{ .Values.supersetNode.livenessProbe.periodSeconds | default 5 }}
+            successThreshold: {{ .Values.supersetNode.livenessProbe.successThreshold | default 1 }}
+          readinessProbe:
+            httpGet:
+              path: /health
+              port: http
+            initialDelaySeconds: {{ .supersetNode.Values.readinessProbe.initialDelaySeconds | default 15 }}
+            timeoutSeconds: {{ .Values.supersetNode.readinessProbe.timeoutSeconds | default 1 }}
+            failureThreshold: {{ .Values.supersetNode.readinessProbe.failureThreshold | default 3 }}
+            periodSeconds: {{ .Values.supersetNode.readinessProbe.periodSeconds | default 5 }}
+            successThreshold: {{ .Values.supersetNode.readinessProbe.successThreshold | default 1 }}
```

Comment:
I think it would be simpler to just set these values and not set a default here. I mean if we have a default it feels the default values should live on `values.yaml` instead of being "hardcoded" on the deployment template

Reply:
I remove default in template and use values.yaml.